src/acme/ui/CompleteATicketUI.java

```diff
@@ -0,0 +1,162 @@
+package acme.ui;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import com.github.lgooddatepicker.components.DateTimePicker;
+import acme.pd.Courier;
+import acme.pd.Ticket;
+import acme.pd.User;
+
+public class CompleteATicketUI extends AcmeBaseJPanel {
+  Ticket ticketToComplete;
+  DateTimePicker dateTimePickerDelivery;
+  DateTimePicker dateTimePickerDeparture;
+  DateTimePicker dateTimePickerPickup;
+  
+
+  public CompleteATicketUI(Ticket ticket) {
+    super();
+    this.ticketToComplete = ticket;
+  }
+
+  public void buildPanel() {
+    // Initialize your panel here
+    this.removeAll();
+    this.setLayout(null);
+    
+    //Keeping with the color scheme
+    Color lightBlue = new Color(93, 184, 202);
+  
+    //title label
+    JLabel lblCompleteTicket = new JLabel("Complete Ticket");
+    lblCompleteTicket.setFont(new Font("Lucida Grande", Font.PLAIN, 24));
+    lblCompleteTicket.setBounds(20, 20, 200, 27);
+    add(lblCompleteTicket);
+    
+    //information section-------------------------------------------
+    JLabel lblPackageId = new JLabel("Package ID");
+    lblPackageId.setBounds(30, 59, 83, 16);
+    add(lblPackageId);
+    
+    JLabel lblCourier = new JLabel("Courier");
+    lblCourier.setBounds(30, 87, 83, 16);
+    add(lblCourier);
+    
+    JLabel lblActPackageId = new JLabel(this.ticketToComplete.getId() + "");
```

Comment:
This should be getPackageID but it may not be in your code, I fleshed it out as a part of create ticket.  If you don't have getPackageID then just put a TODO here to change it later

Reply:
I changed it to package id.